File: cash.py
from cs50 import get_int

while True:
    p = 100 *  float(input("Enter the Amount-$:"))
    if p>0:
      break
coins = 0
while  p >= 25:
    p = p - 25
    coins+= 1

while p>=10:
    p = p - 10
    coins+= 1

while p>=5:
    p = p - 5
    coins+= 1
while p>=1:
    p = p - 1
    coins+= 1

print(coins)

File: 1/cash.c
#include<stdio.h>
#include<math.h>
#include<cs50.h>

int main(void)
{
    float i;
    int a, x, p, y, z, s;
    do
    {
        i = get_float("Enter Amount-$:");
    }
    while (i < 0);
    //rounding to cents

    a = round(i * 100);

    //counting the no. of coins

    for (x = 0; 25 <= a; x++)
    {
        //counting 25 cents

        a = a - 25;

    }
    {
        //counting 10 cents

        for (y = 0; 10 <= a; y++)
        {
            a = a - 10;


        }
        {
            //counting 5 cents

            for (z = 0; 5 <= a; z++)
            {
                a = a - 5;

            }

            {
                //counting 1 cents

                for (p = 0; 1 <= a; p++)
                {
                    a = a - 1;


                }

                {
                    //adding all coins

                    s = x + y + z + p;

                    //printing the no. of coins

                    printf("%i\n", s);

                }

            }


        }

    }

}
